source/package-json.d.ts

```diff
@@ -257,8 +257,26 @@ declare namespace PackageJson {
 		typings?: string;
 	}
 
+	export interface Workspaces {
+		/**
+		The packages blob to use for workspaces.
+		*/
+		packages: string[];
+		/**
+		A list of glob patterns for packages that should not be hoisted.
+		*/
+		nohoist: string[];
+	}
+
 	export interface YarnConfiguration {
 		/**
+    Used to configure yarn (and other) workspaces.
+
+    Please note private should be set to true to use workspaces
```

Comment:
```suggestion
    Please note private should be set to true to use workspaces.
```

And what is "private". Would be good to make it clear you're talking about the top-level private property.